frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantModal/VariantOverrides/VariantOverrides.tsx

```diff
@@ -0,0 +1,163 @@
+import { ChangeEvent, VFC } from 'react';
+import { IconButton, styled, TextField, Tooltip } from '@mui/material';
+import { Delete } from '@mui/icons-material';
+import { Autocomplete } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { InputListField } from 'component/common/InputListField/InputListField';
+import useUnleashContext from 'hooks/api/getters/useUnleashContext/useUnleashContext';
+import { IOverride } from 'interfaces/featureToggle';
+import { OverridesDispatchType } from './useOverrides';
+import SelectMenu from 'component/common/select';
+
+const StyledRow = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    rowGap: theme.spacing(1.5),
+    marginBottom: theme.spacing(2),
+    [theme.breakpoints.down('sm')]: {
+        flexDirection: 'column',
+        '& > div, .MuiInputBase-root': {
+            width: '100%',
+            alignItems: 'flex-start',
+        },
+    },
+}));
+
+const StyledSelectMenu = styled(SelectMenu)(({ theme }) => ({
+    minWidth: theme.spacing(20),
+    marginRight: theme.spacing(10),
+}));
+
+const StyledFieldColumn = styled('div')(({ theme }) => ({
+    width: '100%',
+    gap: theme.spacing(1.5),
+    display: 'flex',
+}));
+
+const StyledInputListField = styled(InputListField)(() => ({
+    width: '100%',
+}));
+
+const StyledTextField = styled(TextField)(() => ({
+    width: '100%',
+}));
+
+interface IOverrideConfigProps {
+    overrides: IOverride[];
+    overridesDispatch: OverridesDispatchType;
+}
+
+export const OverrideConfig: VFC<IOverrideConfigProps> = ({
+    overrides,
+    overridesDispatch,
+}) => {
+    const { context } = useUnleashContext();
+    const contextNames = context.map(({ name }) => ({
+        key: name,
+        label: name,
+    }));
+
+    const updateValues = (index: number) => (values: string[]) => {
+        overridesDispatch({
+            type: 'UPDATE_VALUES_AT',
+            payload: [index, values],
+        });
+    };
+
+    const updateSelectValues =
+        (index: number) => (event: ChangeEvent<unknown>, options: string[]) => {
+            event?.preventDefault();
+            overridesDispatch({
+                type: 'UPDATE_VALUES_AT',
+                payload: [index, options ? options : []],
+            });
+        };
+
+    return (
+        <>
+            {overrides.map((override, index) => {
+                const definition = context.find(
+                    c => c.name === override.contextName
```

Comment:
Please use full parameter names.

Reply:
Addressed in https://github.com/Unleash/unleash/pull/2453/commits/7fee269bda3ff609a4d01bbc12cffda41acd23e8 - Haven't touched it after copying it from the original source: 
https://github.com/Unleash/unleash/blob/7fee269bda3ff609a4d01bbc12cffda41acd23e8/frontend/src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/OverrideConfig/OverrideConfig.tsx#L51-L54